addons/io/org.openhab.io.hueemulation/README.md

```diff
@@ -6,6 +6,10 @@ any Hue compatible application.
 Because Amazon Echo and Google Home control openHAB locally this way, it is a fast and reliable way
 to voice control your installation. See the Troubleshoot section down below though.
 
+## Installation
+
+This Binding is located under the MISC tab.
```

Comment:
This is actually no binding, but an I/O service - that's why it is listed under "System Integrations" at https://www.openhab.org/addons/#io.
All those integrations are under the "MISC" category in the Paper UI, so this is not specific to this add-on (hence we should not add a specific note at this place.